supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -63,108 +63,433 @@ Introduce each release note with a heading that summarizes the release note. Thi
-----
+|Application management |Subscriptions |2.5 |Use GitOps for |See _<insert_link_to_GitOps>_ for more details.
+|===
 
-.Example deprecation notice doc text
-----
-In Red{nbsp}Hat OpenStack Platform (RHOSP) 14, the director graphical user interface is deprecated and is planned to be removed in a future release. Red{nbsp}Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed.
-----
 
-==== Removal notice template
-[subs="+quotes"]
+=== Known issues
+Known issues describe existing problems that customers should be aware of, so that they can mitigate them and avoid unnecessary reporting.
+
+.Known issue engineering template
 ----
-In __<product_name> <current_release>__, __<name of capability or feature>__ has been removed. Bug fixes and support are provided only through the end of the __<previous_release>__ lifecycle. As an alternative to __<name_of_capability_or_feature>__, you can use __<alternative_capability_or_feature_if_available>__ instead.
+Cause - the user action or circumstances that trigger the bug
+Consequence - what the user experience is when the bug occurs
+Workaround - if available
+Result – mandatory if the workaround does not solve the problem completely
 ----
-.Example removal notice doc text
+
+.Known issue release note text
+====
+Heading that summarizes the known issue::
+_<Cause>_. As a consequence, _<consequence>_.
++
+To work around this problem, _<workaround in imperative>_. As a result, _<result>_.
++
+TICKET-REFERENCE
+====
+
+In addition to general style, follow these guidelines:
+
+* Always provide information about a workaround in a separate paragraph:
+** If a workaround exists, describe it in the following format:
++
+To work around this problem, <workaround in imperative>.
+** If no workaround is mentioned, investigate and try to describe how to avoid or partially mitigate the problem. If there is no workaround or mitigation, explicitly say: “No known workaround exists.”
+* Use the present tense.
+* If the known issue applies only to specific batch updates (z-streams), clarify that. For example, the known issue might exist from 4.14.0 to 4.14.4 but not 4.14.5 onwards.
+* Never promise future fixes. Avoid making claims that are related to a future release; do not announce a new component will replace a deprecated one until it is released.
+* For customer reference, include Jira tickets links to all Known issues on the line directly after the entry. Do not place that link inside parenthesis or brackets.
+** If you link to private tickets, inform the user that some Jira tickets might require login credentials, for example: “Some linked Jira tickets are accessible only with Red Hat credentials.” 
```

Comment:
@jafiala we chatted a while back and I thought we decided not to use the word private? See how I have added this to the beginning. Maybe we can use similar language? We say "private" here 3 times. I am happy to change it. :-)

See this: https://github.com/redhat-documentation/supplementary-style-guide/pull/533/files#r2265591358

Reply:
Removed one occurrence of "private" in customer facing text, thanks for bringing this up.